runtime/parachains/src/configuration.rs

```diff
@@ -237,6 +323,9 @@ decl_module! {
 		#[weight = (1_000, DispatchClass::Operational)]
 		pub fn set_group_rotation_frequency(origin, new: T::BlockNumber) -> DispatchResult {
 			ensure_root(origin)?;
+
+			ensure!(new >= 1, Error::<T>::InvalidNewValue);
```

Comment:
to please the CI here and below
```suggestion
			ensure!(!new.is_zero(), Error::<T>::InvalidNewValue);
```